torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -489,6 +532,10 @@ std::vector<at::Tensor> FusionKernelRuntime::runWithInput(
       }
     }
 
+    if (runtimePerfDebug()) {
```

Comment:
Would it make sense to pull this result in constructor of `FusionKernelRuntime`? Depending on the system call implementation, It's probably not too much but could be ~1us range of overhead for a env variable check and a atoi call.

Reply:
I wanted to support running this in a context in python which would make debugging a lot easier in some cases. We may want to do something like only profile a single kernel in backwards even. We could move it from an environment variable at some point, but I would like a global context to play with in case. The output can be quite overwhelming so a constant on or off might be a bit much.